stack.yaml.lock

```diff
@@ -1,12 +0,0 @@
-# This file was autogenerated by Stack.
```

Comment:
Same question asked above. I thought it was generally recommended to check that into source control https://docs.haskellstack.org/en/stable/lock_files/#lock-files

Reply:
Thanks for looking at this @FullstackGJJ. To answer your question, I have been using stack for a long time and have never seen a need to commit that lock file. I know that the idea is to increase reproducility somehow, but I believe it's just not necessary in normal FOSS projects. And when it is committed, I find it always has changes that get in the way of my work. (I don't know the precise rules for when it changes, and don't really want to..)

Reply:
Sounds good. Resolving these comments.